AirCasting/SessionViews/ThresholdAlerts/ThresholdAlertSheet.swift

```diff
@@ -0,0 +1,93 @@
+// Created by Lunar on 01/08/2022.
+//
+
+import SwiftUI
+import AirCastingStyling
+
+struct ThresholdAlertSheet<VM: ThresholdAlertSheetViewModel>: View {
```

Comment:
we resigned from injecting VMs into Views a while ago, now they just create them themselves which is a bit less Generic-awkwardy. We can re-discuss this tho if you want